app/Http/Controllers/ProductController.php

```diff
@@ -49,7 +49,7 @@ public function create()
 
     public function store(ProductRequest $request)
     {
-        $subscription = auth()->user()?->shop?->currentSubscriptions()?->subscription;
+        $subscription = $this->mainShop()?->currentSubscriptions()?->subscription;
         $stores = ProductRepository::query()->where('shop_id', $this->mainShop()->id)->get();
```

Comment:
```suggestion
        $existsProduct = $this->mainShop()->products;
```
eta hole mone hoy valo hoto